src/model/document/document.js

```diff
@@ -455,6 +455,17 @@ class Document {
     }
   }
 
+  authorName( name ){
```

Comment:
So my post-submit code stashes the 'name'  under `doc: { provided: { name: "xyz" }`. If you wanna keep this as a top-level attribute doc, you'll have to update the post-submit field:

https://github.com/PathwayCommons/factoid/blob/3a993ef5b7b6a9922a8f9d4053e29bcca0fdf124/src/client/components/tasks.js#L413

Reply:
@jvwong would this commit (https://github.com/PathwayCommons/factoid/pull/913/commits/4b2c3667d738e9d0ee8a83ca0f90c230cd517ec0) covers what you mean?

Reply:
I agree that the data should be stored in the same way as before, ideally through the same PATCH route.

Reply:
> @jvwong would this commit ([4b2c366](https://github.com/PathwayCommons/factoid/commit/4b2c3667d738e9d0ee8a83ca0f90c230cd517ec0)) covers what you mean?

My doc attribute is `provided.name` rather than `authorName`

Reply:
Okay sorry! I set ``provided.name`` instead of ``provided.authorName``.